quarkus/README.md

```diff
@@ -73,6 +127,9 @@ Latency for GraalVM version:
 ## AWS X-Ray Tracing
 You can add additional detail to your X-Ray tracing by adding a TracingInterceptor to your AWS SDK clients.
 
+Please note that AWS Lambda SnapStart currently does not support X-ray tracing. For this reason, tracing is disabled for all lambda functions in SnapStart version.
+Lambda SnapStart is available in the US East (Ohio, N. Virginia), US West (Oregon), Asia Pacific (Singapore, Sydney, Tokyo), and Europe (Frankfurt, Ireland, Stockholm) Regions.
```

Comment:
You can now refer to the recently added section in the user guide instead of adding the text here. https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html#snapstart-supported-regions

That way you do not have to maintain this line when new regions are supported, and avoid stale data

Reply:
Good idea, will update this section